src/lib/Link.svelte

```diff
@@ -52,6 +52,13 @@
 
   // this will enable `<Link on:click={...} />` calls
   function onClick(e) {
+    // user used a keyboard shortcut to force open link in a new tab
+    if (e.metaKey || e.ctrlKey || e.button !== 0 || e.target.tagName === 'A') {
```

Comment:
The logic here should  be `if (e.target.tagName === 'A' && (e.metaKey || e.ctrlKey || e.button !== 0)) return;` in order to skip those events on links only.

Also i'm not sure if this change can be covered by our e2e tests 🤔

Reply:
Source updated. I also added button as a check and then made it a nested if just as it was a bit more readable and less error prone. I have thoughts about the e2e but I'm going to drop that on the main thread.